app/Http/Requests/Settings/UpdateRequest.php

```diff
@@ -28,6 +28,8 @@ public function rules()
             'enrollments_end_at' => 'required|date|after:enrollments_start_at',
             'exchanges_start_at' => 'required|date|after_or_equal:enrollments_start_at',
             'exchanges_end_at' => 'required|date|after:exchanges_start_at',
+            //'groups_creation_start_at' => 'required|date|after_or_equal:exchanges_end_at',
```

Comment:
@fntneves I left this commented for testing purposes but anyway we need to discuss this, I thought this restriction would fit the feature but want to know your opinion, probably should change "after_or_equal" to "after" though.